### ISSUE TITLE
The library isn't ported to Elm 0.18.

### ISSUE THREAD

**Issue by anacrolix** [roSievers/font-awesome#1]
```
--> There are no versions of roSievers/font-awesome that work with Elm 0.18.0.
    Maybe the maintainer has not updated it yet.
```

**Comment by roSievers** [comment]
Yes, at the moment this project isn't maintained anymore. I'm pretty sure that is the case for all Elm projects without 0.18 support at this point. I suggest you look into the alternate libraries, but I certainly won't mind a fork either. Note that it iirc doesn't support all of the font-awesome features.

I petty much saw no reaction or use of this (your post is the first reply on elm-discuss) and didn't even use it myself anymore. That is why I never bothered porting it. I believe it wouldn't be hard to port it though.

**Comment by anacrolix** [comment]
What are you using instead yourself?

**Comment by roSievers** [comment]
Nothing; I didn't use Icons in any 0.18 project I wrote.
